presto-spark-launcher/pom.xml

```diff
@@ -38,8 +38,14 @@
 
         <dependency>
             <groupId>org.apache.commons</groupId>
-            <artifactId>commons-compress</artifactId>
-            <version>1.19</version>
+            <artifactId>commons-compress</artifactId>        
+            <version>1.26.0</version>
```

Comment:
What's the security issue this fixes and in which version was it fixed? Is there a CVE? If it was fixed before 1.26 I'd prefer to not upgrade past 1.25 just yet, for non-security reasons.

Reply:
FYI, the dependencies in 1.26.0 are borked. If you add this version, you also need to add explicit dependencies on commons-io and commons-codec.